gui/wxpython/gui_core/infobar.py

```diff
@@ -0,0 +1,86 @@
+
+import wx
+import wx.aui
+try:
+    import wx.lib.agw.infobar as IB
+except ImportError:
+    import wx.lib.infobar as IB
+
+
+class InfoBar(IB.InfoBar):
+    """A customized and specialized info bar to used by default"""
+    def __init__(self, parent):
+        IB.InfoBar.__init__(self, parent)
+
+        self._background_color = wx.SystemSettings.GetColour(
+            wx.SYS_COLOUR_HIGHLIGHT
+        )
+        self._foreground_color = wx.SystemSettings.GetColour(
+            wx.SYS_COLOUR_HIGHLIGHTTEXT
```

Comment:
It looks like from your screenshot, the colors have defined alpha and that's why it looks weird. This is specific to your system theme. We should get the color, remove the alpha and use just the RGB component.